notebooks/06_Serving/canary.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: machinelearning.seldon.io/v1
```

Comment:
The filename is kind of confusing, What if I have another canary examples for a different framework in this folder?  Can you rename file to `seldon_canary.yaml`?

Reply:
Right, fix it up.